Studio-Lovelies/GG-JointJustice-Unity
# ActorSlots are now referenced to by name (`Witness`) rather than a one-based index (`1`) inside Ink Stories

**Note: This PR uses [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) for each commit.** (thx @notexplosive!)

## Summary
This branch implements #178 inside 690c15d4b08ad6790bb8a888197f30c3704f7a98.  
This change does **not** impact the game from a player's perspective in any way.
 
Out of convenience, 6a96a805dcbd071fa2c843f5c115d85e25b07111 also addresses various inconsistencies inside files I had open after implementing the feature; they are not directly tied to the feature and isolated to that commit.

## Before/after screenshots and/or animated gif
### Before
```
&SET_ACTOR_POSITION:1,Arin
&JUMP_TO_POSITION:2
```

### After
```
&SET_ACTOR_POSITION:Defense,Arin
&JUMP_TO_POSITION:Witness
```

## Testing instructions
1. Run any Story that was modified in this PR
2. Ensure the scene operates identically to before this PR

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - closes #178
